app/models/katello/concerns/smart_proxy_extensions.rb

```diff
@@ -99,24 +91,8 @@ def self.pulp_primary!
           pulp_primary || fail(_("Could not find a smart proxy with pulp feature."))
         end
 
-        def self.default_capsule
-          pulp_primary
-        end
-
-        def self.default_capsule!
-          pulp_primary!
-        end
-
         def self.with_environment(environment, include_default = false)
```

Comment:
```suggestion
        def self.with_environment(environment)
```

Reply:
This should be fine, I don't see any uses of `include_default = false` with `with_environment()`.